docs/001_develop/03_client-capabilities/021_desktop-interoperability/006_interop_notifications_listener.mdx

```diff
@@ -0,0 +1,224 @@
+keywords: [interaction, interactive, interoperability, interop.io, fdc3, notifications]
+tags:
+- interaction
+- interactive
+- interoperability
+- interop.io
+- notifications
+- fdc3
+- data
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# FDC3 Interop.IO notifications listener
+
+This is a utility element that pipes notifications to [Interop.io notifications](https://docs.interop.io/desktop/reference/javascript/notifications/index.html#overview)
+Use cases:
+* Display notifications in the Interop.io Notifications panel
+
+## Examples
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+Declaration
+```html
+<rapid-interop-notification-listener></rapid-interop-notification-listener>
+```
+
+Usage
+```typescript
+@customElement({
+    name: 'my-element',
+    template: html`
+      <rapid-interop-notification-listener
+        resource-name="ALL_NOTIFY_ALERT_RECORDS"
+        :toastButtons="${(): ToastButton[] => [
+            {
+                condition: ({details}) => details.NOTIFY_SEVERITY !== 'Warning',
+                action: ({details, buttonElement, toastElement}) => {
+                    console.log({details, buttonElement, toastElement});
+                    toastElement.close();
+                },
+                appearance: 'secondary-orange',
+                placeholder: 'Dismiss',
+            },
+        ]}"
+      ></rapid-interop-notification-listener>
+    `,
+})
+export class MyElement extends GenesisElement {}
+```
+</TabItem>
+
+<TabItem value="react">
+Declaration
+```html
+<rapid-interop-notification-listener></rapid-interop-notification-listener>
+```
+
+Usage
+```typescript
+export function MyComponent() {
+
+    if (interopNotificationListenerRef.current) {
+        interopNotificationListenerRef.current.toastButtons = [
+            {
+                condition: ({details}) => details.NOTIFY_SEVERITY !== 'Warning',
+                action: ({details, buttonElement, toastElement}) => {
+                    console.log({details, buttonElement, toastElement});
+                    toastElement.close();
+                },
+                appearance: 'secondary-orange',
+                placeholder: 'Dismiss',
+            },
+        ]
+    }
+
+    return (
+        <rapid-interop-notification-listener
+            ref={interopNotificationListenerRef}
+            resource-name="ALL_NOTIFY_ALERT_RECORDS"
+        >
+        </rapid-interop-notification-listener>
+    );
+}
+```
+</TabItem>
+
+<TabItem value="angular">
+Declaration
+```html
+<rapid-interop-notification-listener></rapid-interop-notification-listener>
+```
+
+Usage
+```typescript
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+@Component({
+    selector: 'my-root',
+    templateUrl: './my.component.html',
+    template: `

```

Comment:
I assume you don't need `templateUrl` in this case?

Reply:
Removed